packages/react-devtools/OVERVIEW.md

```diff
@@ -147,6 +147,23 @@ For example, updating the base duration for a fiber with an id of 1:
 ]
 ```
 
+#### Updating errors and warnings on a Fiber
+
+We record calls to `console.warn` and `console.error` in the backend.
+Periodically we notify the frontend that the number of recorded calls got updated.
+We only send the serialized messages as part of the `inspectElement` event.
+
+
+```js
+[
+  4, // fiber id
+  0, // number of calls to console.error from that fiber
+  3, // number of calls to console.warn from that fiber
+]
+```
+
+
+
```

Comment:
Note to self: Add new REMOVE_ALL operation to this before merging.

Reply:
Added [`b79d1d9` (#20463)](https://github.com/facebook/react/pull/20463/commits/b79d1d998d7ef6da63c6f924af4c18c1956a125e) in case GitHub is already hiding this comment.